helm-chart/renku-gateway/templates/deployment.yaml

```diff
@@ -129,8 +129,17 @@ spec:
               {{ end }}
             - name: GATEWAY_SERVICE_PREFIX
               value: {{ .Values.gatewayServicePrefix | default "/api/" | quote }}
-            - name: GATEWAY_REDIS_HOST
-              value: {{ include "call-nested" (list . "redis" "redis.fullname") }}-master
+            - name: REDIS_HOST
+              value: {{ .Values.global.redis.host | default (include "call-nested" (list . "redis" "redis.fullname")) }}
```

Comment:
I think we should not compose the name here. We are accessing `.Values.global.redis.<something>` for all the other env variables, should we keep it coherent and set the name in `.Values.global.redis.host` ?

Reply:
See the discussion on the renku repo about this. However, I removed the now obsolete ugly "call-nested hack" to make this line a bit simpler.

Reply:
I stand corrected. The hack for this is still needed: https://github.com/helm/helm/issues/4535